src/test/java/BaseTest.java

```diff
@@ -70,6 +81,87 @@ public static void provideEmail(String email) {
         emailField.sendKeys(email);
     }
 
+    public static void homePage(){
+        WebElement homePage = driver.findElement(By.xpath("//a[contains(text(),'Home')]"));
+        homePage.click();
+    }
+    public static void playLists(){
+        WebElement playLists = driver.findElement(By.xpath("//h1[contains(text(),'Playlists')]"));
+        playLists.isDisplayed();
+    }
+    public static void choosePlaylist(){
+<<<<<<< Updated upstream
```

Comment:
Please remove this lines

Reply:
After commit & push, it always have this line "<<<<<Updated upstream". Also, do I need to remove "playlists.isDisplaed();" & "public static void choosePlaylist(){" ?